server/test/User/updateResume.js

```diff
@@ -31,7 +31,7 @@ export default updateResumeTest = () => {
 
       expect(response.status).toBe(201);
       expect(response.body.data._id).toBe(userId.toString());
-      expect(response.body.data.resume_url).toBe(resumeLink);
+      expect(response.body.data.resume.url).toBe(resumeLink);
```

Comment:
Your unit test implementation are ok here, but not the best structure. Unit test should be as self-contained as possible. Therefore, we want to avoid making network calls whenever possible. Take a look at mocking using Jest.